test/iht/views/application/assets/insurancePolicy/InsurancePolicyDetailsMoreThanMaxValueViewTest.scala

```diff
@@ -45,7 +46,9 @@ class InsurancePolicyDetailsMoreThanMaxValueViewTest extends YesNoQuestionViewBe
     form => insurance_policy_details_more_than_max_value(form, regDetails)
 
   override def cancelComponent = Some(CancelComponent(routes.InsurancePolicyOverviewController.onPageLoad(),
-    messagesApi("site.link.return.insurance.policies")))
+    messagesApi("site.link.return.insurance.policies"),
```

Comment:
I'd probably not do it this way. Instead I'd create a method to append the fragment id onto the end of the url and pass the result of this into the original CancelComponent, which then only needs two parameters. However, the effort involved in changing it at this stage may not be worth it. We can discuss it when you get in.